CMakeLists.txt

```diff
@@ -16,6 +16,8 @@ find_package(catkin REQUIRED
 add_message_files(
   FILES
   RGBState.msg
+  BinaryState.msg
+  GrayScale.msg
```

Comment:
Do we need both a grayscale and a binary messages? How are they used differently?